ci-operator/step-registry/mirror-images/by-oc-mirror/mirror-images-by-oc-mirror-commands.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export HOME="${HOME:-/tmp/home}"
+export XDG_RUNTIME_DIR="${HOME}/run"
+export REGISTRY_AUTH_PREFERENCE=podman # TODO: remove later, used for migrating oc from docker to podman
+mkdir -p "${XDG_RUNTIME_DIR}"
+
+mirror_output="${SHARED_DIR}/mirror_output"
+install_config_icsp_patch="${SHARED_DIR}/install-config-icsp.yaml.patch"
+icsp_file="${SHARED_DIR}/local_registry_icsp_file.yaml"
+image_set_config="${SHARED_DIR}/image_set_config.yaml"
+
+# private mirror registry host
+# <public_dns>:<port>
+MIRROR_REGISTRY_HOST=`head -n 1 "${SHARED_DIR}/mirror_registry_url"`
+if [ ! -f "${SHARED_DIR}/mirror_registry_url" ]; then
+    echo "File ${SHARED_DIR}/mirror_registry_url does not exist."
+    exit 1
+fi
+echo "MIRROR_REGISTRY_HOST: $MIRROR_REGISTRY_HOST"
+echo "OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
+
+# check whether nightly payload.
+if echo $OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE |grep "nightly"; then
```

Comment:
In prow ci, it is using a digest format of image, there is not tag name to indicate it is nightly image.

Reply:
updated the code, thanks